### PR TITLE
Fix/rogue detection

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Added all the ip of the server in the list of dhcp server to avoid false positive rogue dhcp detection

# Impacts
No

# Delete branch after merge
YES

# NEWS file entries

## Bug Fixes
* False positive dhcp rogue detection

### CODE REVIEW

**Comment by veedubbs** [comment]
I'm very interested in this patch.  I'm getting spammed with rogue detection emails right now; about 60 - 70 messages per hour.

**Comment by fdurand** [comment]
BUMP

File: lib/pf/dhcp/processor.pm [hunk 1]
```diff
@@ -81,8 +82,12 @@ Returns an hash of arrays
 sub _get_local_dhcp_servers {
     # Look for local DHCP servers by IP if not already existent in local cache and fill it up
     unless ( @local_dhcp_servers_ip ) {
-        foreach my $network ( keys %ConfigNetworks ) {
-            push @local_dhcp_servers_ip, $ConfigNetworks{$network}{'gateway'} if ($ConfigNetworks{$network}{'dhcpd'} eq 'enabled');
+        foreach my $network ( keys %NetworkConfig ) {
+            if ($NetworkConfig{$network}{'dhcpd'} eq 'enabled') {
+                push @local_dhcp_servers_ip, $NetworkConfig{$network}{'gateway'};
+		push @local_dhcp_servers_ip, $NetworkConfig{$network}{'vip'} if ($NetworkConfig{$network}{'vip'});
```
Comment (julsemaan) [inline lib/pf/dhcp/processor.pm]: indentation issue

**Comment by julsemaan** [comment]
Just fix the indentation issue and we're good to merge

**Comment by fdurand** [comment]
Bump

**Pull request merged**